test/e2e/performanceprofile/testdata/render-expected-output/manual_node.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: config.openshift.io/v1
+kind: Node
```

Comment:
Kind: Node? Shouldn't that be NodeConfig?

Reply:
Btw, we have a makefile target to render and update the expected manifests (make render-sync iirc).

Reply:
render already mentioned here: https://github.com/openshift/cluster-node-tuning-operator/pull/733#issuecomment-1651969605